docs/contribute/index.md

```diff
@@ -28,6 +27,16 @@ to install.
 You may contribute to any of the core packages or develop your own module as a stand-alone package (see below).    
 
 
+## Style Guide
+
```

Comment:
Maybe have two subsections to make this clearer:
### Code

### Documentation

Reply:
good idea!